### ISSUE TITLE
YouTubePlayerView - white screen before onReady is called

### ISSUE THREAD

**Issue by asbozh** [PierfrancescoSoffritti/android-youtube-player#817]
It seems that the YouTubePlayerView background in version 10.0.5 is black but in the latest version is changed to white?

Even in the sample app in a lot of examples there is a white background of the view until the onReady callback is called and the video starts loading.

For me it seems as regression and I cannot find a way to adjust it

https://user-images.githubusercontent.com/12729014/153768561-874cacda-10e5-4582-b954-d5249a768f10.mov

.

**Comment by PierfrancescoSoffritti** [comment]
Hi, thanks for reporting the issue, I will look into it. In the meantime I think you can just add a black view behind the player?

**Comment by asbozh** [comment]
Yes, I thought of the same workaround with adding black overlay view. But in my case the YouTubePlayerView is in Constraint Layout and has layout_width of 0dp and layout_height of wrap_content and to put the overlay view - I have to match the same dimensions of the YouTubePlayerView which are based on the display size and here is the issue as for the overlay view the wrap_content height is hard to calculate.

**Comment by PierfrancescoSoffritti** [comment]
In constraint layout, put the constraints of the black view on the YouTubePlayerView edges.

**Comment by PierfrancescoSoffritti** [comment]
Can you provide a video/screenshot of the issue?

**Comment by Surkhojb** [comment]
Hi @PierfrancescoSoffritti I managed to fix it making the player visible once is ready 
`override fun onReady(youTubePlayer: YouTubePlayer) {
                    super.onReady(youTubePlayer)
                    binding.player.visible()
                    player = youTubePlayer
                    youTubePlayer.cueVideo(videoId, 0f)
                }`

**Comment by PierfrancescoSoffritti** [comment]
Thanks for sharing!
